core/storage/user/gae_models.py

```diff
@@ -296,6 +308,23 @@ class UserSubscriptionsModel(base_models.BaseModel):
     # When the user last checked notifications. May be None.
     last_checked = ndb.DateTimeProperty(default=None)
 
+    @staticmethod
+    def export_data(user_id):
+        """Export UserSubscriptionsModel data."""
+        user_model = UserSubscriptionsModel.get(user_id, strict=False)
+
+        if not user_model:
+            user_model = UserSubscriptionsModel(id=user_id)
+
+        user_data = dict()
```

Comment:
You can do:

    user_data = {
        'activities_ids': ...,
        ...
    }

Shouldn't it be activity_ids, btw?